db/migrate/20200622134955_remove_service_columns.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "system/database/#{System::Database.adapter}"
+
+class RemoveServiceColumns < ActiveRecord::Migration[5.0]
+  def up
+    columns = %w[act_as_product end_user_registration_required default_end_user_plan_id
+      oneline_description txt_api txt_features infobar draft_name display_provider_keys
+      credit_card_support_email admin_support_email tech_support_email
+    ]
+
+    sql_remove_columns = if System::Database.oracle?
+      "ALTER TABLE services DROP (#{columns.join(', ')})"
```

Comment:
Why not using directly `remove_columns`?
Difference is that it will do in multiple statements ... is there any benefit doing one single LATER?

Reply:
yes, it is MUCH faster doing it like this.
What you are suggesting is one of the first things that I tested.

Reply:
More or less what it takes for each table of multiple columns individually, it is what it would take for each of the columns of the table if done that way.

Reply:
Look for example at the services table in https://github.com/3scale/porta/pull/1979#issuecomment-651090774